.env

```diff
@@ -0,0 +1,16 @@
+#Server Information
```

Comment:
Ideally, the `.env` file should be added to `.gitignore`.

You can instead add a `.env.sample` file and add instructions in the `README.md` to create this file.

Reply:
I thought about it. But as for now we are not having any API SECRET or KEY stored in .env so I didn't add it to .gitignore.
Thanks